modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -666,6 +672,26 @@ private void _createTable(
 		runSQL(dynamicObjectDefinitionTable.getCreateTableSQL());
 	}
 
+	private void _deleteObjectEntriesWorkflowTasks(
```

Comment:
Hey @mateussgn , Would you mind to change the name of this method to _deleteWorkflowInstanceLinks?